.github/workflows/e2e.yml

```diff
@@ -4,28 +4,13 @@ on:
   push:
     branches:
       - master
+      - refactor-add-ie-e2e-test
```

Comment:
Pls remove the branch before merging this branch because the only purpose is to run GitHub Actions on this branch🙏